rxbinding-support-recyclerview/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.jakewharton.rxbinding.support.v7"
```

Comment:
It's not the class package, but the package defined in the library's manifest (where it's R file is generated). For RecyclerView it's https://developer.android.com/reference/android/support/v7/recyclerview/package-summary.html